src/html/partials/key-factors.html

```diff
@@ -3,37 +3,46 @@
   <h2 class="sm-title">Why People Don't Adhere</h2>
   <p class="lg-text">A myriad of factors conspire to prevent people from sticking to their treatment programs. But a few are most telling.</p>
 
-  <!-- <div class="factors-section__human-figure-wrapper">
-    <div class="factors-section__human-figure-gear"></div>
-    <div class="factors-section__human-figure-heart"></div>
-    <div class="factors-section__human-figure-dollar"></div>
-    <div class="factors-section__human-figure-tally"></div>
+  <!-- <div class="factors-section__human-figure">
+    <div class="factors-section__human-figure__gear"></div>
+    <div class="factors-section__human-figure__heart"></div>
+    <div class="factors-section__human-figure__dollar"></div>
+    <div class="factors-section__human-figure__tally"></div>
   </div> -->
 
+  <!-- for <h3> with orange text, I'm adding class '-orange' and deviating slightly from BEM -->
+
   <ul class="factors-section__list">
     <li class="factors-section__list__item">
-      <!-- <div class="factors-section__gear"></div> -->
-      <h3>Comprehension</h3>
-      <p>“All of that medical jargon doesn’t make much sense to me but I don’t want to bother the doctor with my silly
-        questions.”</p>
+      <button class="factors-section__list__item__btn--gear"></button>
+      <div class="factors-section__list__item__wrapper">
+        <h3 class="sm-list-title -orange">Comprehension</h3>
+        <p class="sm-sans-serif-text">“All of that medical jargon doesn’t make much sense to me but I don’t want to bother the doctor with my silly questions.”</p>
+      </div>
     </li>
 
     <li class="factors-section__list__item">
-      <!-- <div class="factors-section__heart"></div> -->
-      <h3>Denial</h3>
-      <p>“I feel just fine. I don’t know why that doctor thinks I’m sick. Why would I take medicine that I don’t need?”</p>
+      <button class="factors-section__list__item__btn--heart"></button>
+      <div class="factors-section__list__item__wrapper">
+        <h3 class="sm-list-title -orange">Denial</h3>
+        <p class="sm-sans-serif-text">“I feel just fine. I don’t know why that doctor thinks I’m sick. Why would I take medicine that I don’t need?”</p>
+      </div>
     </li>
 
     <li class="factors-section__list__item">
-      <!-- <div class="factors-section__dollar"></div> -->
-      <h3>Cost</h3>
-      <p>“I took my prescription to the pharmacy but they said it would cost several hundred dollars! I just couldn’t afford it.”</p>
+      <button class="factors-section__list__item__btn--dollar"></button>
+      <div class="factors-section__list__item__wrapper">
+        <h3 class="sm-list-title -orange">Cost</h3>
+        <p class="sm-sans-serif-text">“I took my prescription to the pharmacy but they said it would cost several hundred dollars! I just couldn’t afford it.”</p>
+      </div>
     </li>
 
     <li class="factors-section__list__item">
-      <!-- <div class="factors-section__tally"></div> -->
```

Comment:
I think you can remove these factors section icons, since they got moved above.

Reply:
Are you referring to the commented out lines 6-11?

Reply:
No, just the individual ones here in each `<li>`.

Reply:
I turned them into `<buttons>`, are you looking at the `wrapper`?